### PR TITLE
Developed intro website for component 

### PR DESCRIPTION
**Pull request by rahulcs754**
- use navigation in the intro website 
- add extra utility class
- add and remove text 
- created define component page 
- Added some data into readme file about component lib and links
**
image
**

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
check for unncessary space ! remaing looks good to me

File: css/utils/utilities.css [hunk 1]
```diff
@@ -205,7 +245,7 @@
 }
 
 .font-s {
```
Comment (coldpigli) [inline css/utils/utilities.css]: Could have used a better classname for this.

File: docs/intro/pages/list.html [hunk 2]
```diff
@@ -327,42 +328,47 @@ <h3>Checkbox List</h3>
 
 
 
-
```
Comment (coldpigli) [inline docs/intro/pages/list.html]: A lot of redundant spaces here

File: docs/intro/pages/list.html [hunk 3]
```diff
@@ -327,42 +328,47 @@ <h3>Checkbox List</h3>
...
-      <span>  
-          
-      </span> 
-       <span> Connect  - <a href="https://www.linkedin.com/in/rahulcs754/" target="_blank"><i class="fab fa-linkedin connect-link"></i></a> | <a href="https://github.com/rahulcs754" target="_blank"><i class="fab fa-github connect-link"></i></a>  |  <a href="https://twitter.com/Rahulcs754" target="_blank" ><i class="fab fa-twitter connect-link"></i></a>  Rahul Shukla</span>
+        <span>
+
+        </span>
+        <span> Connect - <a href="https://www.linkedin.com/in/rahulcs754/" target="_blank"><i
+                    class="fab fa-linkedin connect-link"></i></a> | <a href="https://github.com/rahulcs754"
+                target="_blank"><i class="fab fa-github connect-link"></i></a> | <a
```
Comment (coldpigli) [inline docs/intro/pages/list.html]: Could be split into different lines for better readability

**Review by coldpigli — COMMENTED** [review]
Added a few suggestions.

File: css/utils/utilities.css (hunk 1 above)
Comment (rahulcs754) [reply css/utils/utilities.css]: as you suggested to me. i changed
image

Comment (rahulcs754) [reply css/utils/utilities.css]: image
I changed the class name as you suggested now it is meaning full.
Thanks for giving suggestion

File: docs/intro/pages/list.html (hunk 2 above)
Comment (rahulcs754) [reply docs/intro/pages/list.html]: removed space

File: docs/intro/pages/list.html (hunk 3 above)
Comment (rahulcs754) [reply docs/intro/pages/list.html]: done this changes to all file

**Review by coldpigli — APPROVED** [review]

**Pull request merged**